.github/scripts/generate-coverage.py

```diff
@@ -211,20 +243,26 @@ def buildQueries(language: str):
 
     all_languages = False
     languages = []
+    codeql_binary = findCodeQL()
+
     if not arguments.language:
-        languages = ["cpp", "csharp", "java", "javascript", "python", "ruby"]
+        languages = language_display.keys()
         all_languages = True
     else:
         languages = [arguments.language]
```

Comment:
I think you should check if arguments.language is in those we support, to validate the input. You can do that here, or in the `add_argument()` call